### DISCUSSION TITLE
[World Bosses] Change the phasing/loot reset from 72h to 70h

### DISCUSSION THREAD

**Discussion by anhility** [chromiecraft/chromiecraft#3773] [Suggestions]
Due to the lockout for each world boss happens on the kill and not on a predetermined time or date,
this can cause the time when you can pull the boss to become later and later if done back to back every three days.

My suggestion to lower the reset from 72h to 70h.
- It will prevent the accumulated time lag that would build up.
- This will make raid scheduling easier as you don't have to keep the time of kill in mind.
- Would be similar with how profession cooldowns work as they also aren't fully 24h or 72h.
- It would still, in practice, be every three days you are able to get loot from them.

**Comment by Annamaria-CC** [comment]
I like this idea
